src/evm-instructions.h

```diff
@@ -26,6 +26,77 @@
 
 namespace HeraVM {
 
-const std::map<uint8_t, std::string> evmInstructionNames;
-
+const std::map<uint8_t, std::string> evmInstructionNames = {
```

Comment:
The actual values are in the `.cpp` file :)

Reply:
If the map is declared const in the header, does that mean it can be reassigned in the cpp file?

When I use this PR with tracing, `evmInstructionNames.size()` is 0.

Reply:
I moved the map into `evm-instructions.h` and removed `evm-instructions.cpp` as this seemed to be the only way to keep the map `const`.

Reply:
The problem with having it in a header it cannot be included in multiple files, unless marked `static` and in that case it is duplicated in the bytecode.

Reply:
Ah okay.

Reply:
@axic I got it working by adding `extern` to the header declaration.